bessctl/commands.py

```diff
@@ -784,11 +784,15 @@ class Foo(Module):
 
 # NOTE: the name of this function is used below
 def _do_run_file(cli, conf_file):
-    if not os.path.exists(conf_file):
+    if not os.path.isfile(conf_file):
         cli.err('Cannot open file "%s"' % conf_file)
         return
 
-    xformed = sugar.xform_file(conf_file)
+    try:
+        xformed = sugar.xform_file(conf_file)
+    except:
```

Comment:
Should probably list the specific exceptions here, although it's annoying since Python3 uses `FileNotFoundError` if it doesn't find the file, while py2k uses `IOError` (and doesn't have `FileNotFoundError`!).  We probably shouldn't use `os.path.exists` or `os.path.isfile` at all—just try the `io.open`.  Maybe we should put the exception handling in `sugar.py` itself since the testing code uses it too?